contents/Documentation/Optics.playground/Pages/Overview.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,54 @@
+// nef:begin:header
+/*
+ layout: docs
+ title: Overview
+ */
+// nef:end
+/*:
+ # Overview
+ 
+ {:.beginner}
+ beginner
+ 
+ The optics module in Bow provides several utilities to work with immutable data structures. It enables easy access to getting, setting and modifying deeply nested data structure without the burden of making explicit copies of the involved values.
+ 
+ In order to use the module, you just need to import it:
+ */
+import BowOptics
+/*:
+ This module includes multiple facilities to work with immutable nested data structures:
+ 
+ - Optics: it includes several data types that let us work with different types of structures.
+ - Type classes: several abstractions are provided to generalize operations of accessing data structures internals.
+ - Automatic generation: in some cases, the library is able to automatically provide certain optics for some data types, so that you do not need to write them.
+ 
+ ## Optics
+ 
+ The provided optics operate on two type arguments that are typically named `S` and `A`, which correspond to the source (the whole structure we have) and the focus (the part we want to obtain or modify). The following table summarizes the individual optics that are provided in `BowOptics`:
+ 
+ | Optic     | Description |
+ | --------- | ----------- |
+ | Iso       | A lossless invertible optic defining an isomorphism between two types |
+ | Getter    | An optic that can focus into a structure and get its focus |
+ | Setter    | An optic that can focus into a structure and set or modify its focus |
+ | Lens      | An optic that can focus into a structure and get, set or modify its focus |
+ | Optional  | An optic whose focus is optional and can get, set or modify it |
+ | Prism     | An optic whose focus is present under certain circumstances and can get, set or modify it |
```

Comment:
I think `Prism` has a lazy explain - `under certain circumstances` - as you did in `Lens` maybe you can add these kinds of lens apply into a sum data types :)